### ISSUE TITLE
Press TAB to restart an active test.

### ISSUE THREAD

**Issue by ryan-harrington** [brancobruyneel/crabtyper#4]
The <kbd>n</kbd> key is used to skip the currently displayed snippet while the test is inactive, but there doesn't seem to be any shortcut to restart a test while it is active (<kbd>CTRL</kbd>+<kbd>R</kbd> works but is less convenient). This shortcut can be used whether the test is active or inactive. 

There is currently no use for the <kbd>TAB</kbd> key while typing the snippets as white space is skipped over. If this happens to change in the future, the restart key may have to be something like <kbd>ESC</kbd> or another preferred shortcut.

**Comment by brancobruyneel** [comment]
Good point, although I like the way of going to normal mode (which adheres more to Vim motions). I can see that pressing the `TAB` key is more convenient for most people.
